aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -647,6 +663,24 @@ func resourceAwsKinesisFirehoseDeliveryStream() *schema.Resource {
 			},
 		},
 
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(20 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(20 * time.Minute),
+		},
+
+		CustomizeDiff: customdiff.Sequence(
```

Comment:
We should prefer to avoid `CustomizeDiff` in almost all situations. There are numerous issues with its implementation in the Terraform Provider SDK when coupled with other Terraform functionality, such as `ignore_changes`. It can be removed with the `DiffSuppressFunc` proposed in the schema.

Reply:
Done.